include/LightGBM/dataset.h

```diff
@@ -24,6 +26,19 @@
 
 namespace LightGBM {
 
+class Str2Num {
```

Comment:
>class [](http://example.com/codeflow?start=0&length=5)

since all of the members are public, so how about change it to structure instead of class?